Update troubleshooting doc to fix PVC pending issue on Amazon EKS 1.23 + (#479)
kubecost/docs: troubleshoot-install.md

    --set prometheus.server.persistentVolume.enabled="false"
    ```

## Issue: waiting for a volume to be created, either by external provisioner "ebs.csi.aws.com" or manually created by system administrator.

> If the PVC is in pending state for more than 5 minutes, the cluster is Amazon EKS 1.23+ the error message appears as the following example:

Reviewer: Should this be "and the"?
```suggestion
If the PVC is in a pending state for more than 5 minutes, and the cluster is Amazon EKS 1.23+ the error message appears as the following example:
```

```bash
kubectl describe pvc cost-analyzer -n kubecost | grep "ebs.csi.aws.com"

> ```

Reviewer: Add blank line between code block and text.

> Example result

Reviewer: ```suggestion
Example result:
```

> ```

Reviewer: Add blank line between code block and text.
Add language to code block

```suggestion

``` bash
```

Annotations:   volume.beta.kubernetes.io/storage-provisioner: ebs.csi.aws.com
               volume.kubernetes.io/storage-provisioner: ebs.csi.aws.com
  Normal  ExternalProvisioning  69s (x82 over 21m)  persistentvolume-controller  waiting for a volume to be created, either by external provisioner "ebs.csi.aws.com" or manually created by system administrator
```

> You need to install [AWS EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) because the Amazon EKS cluster version 1.23+ uses "ebs.csi.aws.com" provisioner even the AWS EBS CSI ddriver has not been installed yet.

Reviewer: ```suggestion
You need to install the [AWS EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) because the Amazon EKS cluster version 1.23+ uses "ebs.csi.aws.com" provisioner and the AWS EBS CSI driver has not been installed yet.
```
